app/views/application_mailer/sample_email.html.erb

```diff
@@ -9,7 +9,9 @@
       Sample mail sent using smtp.
     </p>
     <% @user.posts.each do |user_post| %>
```

Comment:
Instead of iterating on all the post why don't you iterate only on pending tasks and avoid conditional checks on your view? You alredy have a scope for that